flight/src/server.rs

```diff
@@ -96,6 +96,32 @@ impl<T> FeedTable<T> {
                 Field::new("stream_cid", DataType::Binary, false),
                 Field::new("stream_type", DataType::UInt8, false),
                 Field::new("controller", DataType::Utf8, false),
+                Field::new(
+                    "dimensions",
+                    DataType::Map(
+                        Field::new(
+                            "entries",
+                            DataType::Struct(
+                                vec![
+                                    Field::new("key", DataType::Utf8, false),
+                                    Field::new(
+                                        "value",
+                                        DataType::Dictionary(
+                                            Box::new(DataType::Int8),
```

Comment:
Out of curiosity, will we support any other types? I'm curious why you choose Int8 instead of 32 or 64 and just binary strings instead of utf8.

Reply:
Int8 is about how many unique values we can have. Int8 gives us 256. And as I write this I realize this is not enough. I was thinking about the number of unique dimension values for a single stream. However this needs to cover many streams combined. I'll likely pick i32/i64 as its plenty large.